recipe/meta.yaml

```diff
@@ -145,45 +135,6 @@ outputs:
         - pip check
       script: test-py-xgboost.py
 
-  - name: py-xgboost-cpu
-    build:
-      noarch: python
-      skip: true  # [cuda_compiler != "None"]
-    requirements:
-      host:
-        - python {{ python_min }}
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-        - py-xgboost =*={{ string_prefix }}_pyh*_{{ PKG_BUILDNUM }}
-      run:
-        - python >={{ python_min }}
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-        - py-xgboost =*={{ string_prefix }}_pyh*_{{ PKG_BUILDNUM }}
-    test:
-      requires:
-        - python {{ python_min }}
-      imports:
-        - xgboost
-
-  - name: py-xgboost-gpu
-    build:
-      noarch: python
-      skip: true  # [cuda_compiler == "None"]
-    requirements:
-      host:
-        - python {{ python_min }}
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-        - py-xgboost =*={{ string_prefix }}_pyh*_{{ PKG_BUILDNUM }}
-      run:
-        - python >={{ python_min }}
-        - {{ pin_subpackage('py-xgboost', exact=True) }}
-        - py-xgboost =*={{ string_prefix }}_pyh*_{{ PKG_BUILDNUM }}
-        - __cuda  # [cuda_compiler != "None"]
-    test:
-      requires:
-        - python {{ python_min }}
-      imports:
-        - xgboost
-
   - name: xgboost
     build:
       noarch: python
```

Comment:
Though we also have `xgboost`, which is essentially an alias for `py-xgboost`

This is also matches the name on PyPI